tests/letters_test.js

```diff
@@ -57,8 +57,7 @@ test("create, list, read, then cancel, a letter with no extra services", async f
         to: to,
         from: from,
         color: true,
-        file:
```

Comment:
Not sure what's up with `prettier`, but these changes fail `prettier` in CD. :sob: